jobs/build/check-bugs/Jenkinsfile

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env groovy
+node {
+    checkout scm
+    buildlib = load("pipeline-scripts/buildlib.groovy")
+    commonlib = buildlib.commonlib
+    commonlib.describeJob("check-bugs", """
+        ----------
+        Check Bugs
+        ----------
+        Looks for blocker bugs and potential regressions, report findings on Slack.
+
+        Timing: Daily run, scheduled.
+    """)
+}
+
+pipeline {
+    agent any
+
+    options {
+        disableResume()
+        skipDefaultCheckout()
+    }
+
+    parameters {
+        choice(
+            name: "BUILD_VERSION",
+            description: "OSE Version",
+            choices: commonlib.ocpMajorVersions['all'],
+        )
+    }
+
+    stages {
+        stage("check-blockers") {
+            steps {
+                script {
+                    blocker_bugs = commonlib.shell(
+                        returnStdout: true,
+                        script: """
+                            ${buildlib.ELLIOTT_BIN}
+                            --group openshift-${params.BUILD_VERSION}
+                            find-bugs
+                            --mode blocker
+                            --report
+                        """.stripIndent().tr("\n", " ").trim()
+                    ).trim()
+                }
+            }
+        }
+        stage("check-regressions") {
+            steps {
+                script {
+                    if (next_is_prerelease(params.BUILD_VERSION)) {
+                        return
+                    }
+                    bugs = commonlib.shell(
+                        returnStdout: true,
+                        script: """
+                            ${buildlib.ELLIOTT_BIN}
+                            --group openshift-${params.BUILD_VERSION}
+                            find-bugs
+                            --mode sweep
+                            | tail -n1
+                            | cut -d':' -f2
+                            | tr -d ,
+                        """.stripIndent().tr("\n", " ").trim()
+                    ).trim()
+
+                    potential_regressions = commonlib.shell(
+                        returnStdout: true,
+                        script: """
+                            ${buildlib.ELLIOTT_BIN}
+                            --group openshift-${params.BUILD_VERSION}
+                            verify-bugs ${bugs}
+                        """.stripIndent().tr("\n", " ").trim()
+                    ).trim()
+                }
+            }
+        }
+        stage("slack-notification") {
```

Comment:
It'd be nice if the text would be a bit nicer:
- Special content when no issues were found (green check mark or so)
- only display `blocker_bugs` if that test has something to look at (no header for things we don't need to look at)
- Have clickable links on the bugs, not only text of their ID